src/shogun/classifier/mkl/MKLClassification.cpp

```diff
@@ -38,3 +38,16 @@ void CMKLClassification::init_training()
 	REQUIRE(m_labels->get_num_labels(), "Number of labels is zero.\n");
 	REQUIRE(m_labels->get_label_type() == LT_BINARY, "Labels must be binary.\n");
 }
+
+CMKLClassification* CMKLClassification::obtain_from_generic(CMachine* machine)
```

Comment:
one issue I see is that the method declaration cannot be generated since it has to be in the header